harness/determined/common/experimental/determined.py

```diff
@@ -74,7 +74,10 @@ def create_user(
     ) -> user.User:
         create_user = bindings.v1User(username=username, admin=admin, active=True, remote=remote)
         hashedPassword = None
-        if password is not None:
+        if not remote:
+            if password is None:
```

Comment:
What keeps you from putting this complexity requirement and message into `check_password_complexity`? Or, should we go that route, sending it to the backend without a password and waiting for its approval/rejection?